### PR TITLE
footer component

### LINKED ISSUE #34: Footer Component
## Common Component - Footer :
The user is going to have a footer on every page and the footer contains a navigation bar to social media links, Also the website logo.
- props: color, position.

- [ ]  It will be used in Pages :
almost every page.

- [ ]  Material UI link :
[Footer link](https://material-ui.com/components/bottom-navigation/#bottom-navigation-2)

- [ ] UI Image From Design :

image

### PR DESCRIPTION
**Pull request by imansedky5**
fix #34

### CODE REVIEW

**Review by mFlaifel — COMMENTED** [review]
- fix console errors
- remove margin-top
-

**Review by khamis-hesham — DISMISSED** [review]

**Review by AdhamHaisami — APPROVED** [review]

File: client/src/component/footer/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import useStyles from './style';
+
+const Footer = () => {
+  const classes = useStyles();
+  return (
+    <Grid container className={classes.footer}>
+      <div className={classes.imgContainer}>
+        <img alt="girl" src={logo} className={classes.img} />
+      </div>
+
```
Comment (mFlaifel) [inline client/src/component/footer/index.js:14]: remove this empty line
and try to format this file with prettier

File: client/src/component/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <Typography className={classes.typo}>
+        @2021 All Rights Reserved
```
Comment (mFlaifel) [inline client/src/component/footer/index.js:16]: make year dynamic using date object

**Review by khamis-hesham — APPROVED** [review]

**Review by mFlaifel — APPROVED** [review]
well done

**Review by AdhamHaisami — APPROVED** [review]
Great!!

**Review by yasmeenattallah — APPROVED** [review]

**Pull request merged**